devices/MiHumidifierMJJSQL.js

```diff
@@ -0,0 +1,79 @@
+// deerma.humidifier.mjjsq contract: https://github.com/rytilahti/python-miio/issues/533, https://github.com/rytilahti/python-miio/blob/master/miio/airhumidifier_mjjsq.py
+const MiHumidifierCBL = require('./MiHumidifierCBL');
+
+module.exports = class extends MiHumidifierCBL {
+    constructor() {
+        super();
+        this.version = "mjjsql";
+    }
+
+    initializePower() {
+        super.initializePower();
+        this.powerGetName = "OnOff_State";
+        this.convertPowerToActivity = function (power) {
+            return power === 1 ? Characteristic.Active.ACTIVE : Characteristic.Active.INACTIVE;
```

Comment:
Here and in all similar functions `Characteristic` will be undefined.

Reply:
I've got the following error when tried to start homebridge:

```
getCurrentHumidifierState ReferenceError: Characteristic is not defined
    at module.exports.convertPowerToHumidifierState (/opt/homebridge/npm-global/lib/node_modules/homebridge-mi-humidifier/devices/MiHumidifierV1.js:20:101)
    at MiHumidifier.getCurrentHumidifierState (/opt/homebridge/npm-global/lib/node_modules/homebridge-mi-humidifier/index.js:223:43)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Reply:
Fixed